src/test/resources/database/add-category-to-categories-table.sql

```diff
@@ -0,0 +1 @@
+insert into categories (id, name, description) values (1, "Horror", "Horror type");
```

Comment:
Add one empty line at the end of the file (check it everywhere) 
```suggestion
INSERT INTO categories (id, name, description) VALUES (1, "Horror", "Horror type");
```
Use the same SQL formatting for each of your scripts.